src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java

```diff
@@ -538,6 +538,16 @@ protected boolean handleResponse(BulkItemResponse response,
                   response.getVersion(),
                   response.getIndex()
           );
+
+          log.trace("{} version conflict for operation {} on document '{}' version {}"
+                          + " in index '{}' and stacktrace '{}'",
+                  request != null ? request.versionType() : "UNKNOWN",
+                  response.getOpType(),
+                  response.getId(),
+                  response.getVersion(),
+                  response.getIndex(),
+                  response.getFailure().getCause().getStackTrace()
```

Comment:
Are we sure that getFailure won't be null here. Better to just print `response.getFailure()` instead of `response.getFailure().getCause().getStackTrace()` and change the log from `stacktrace` to `failures`.

Reply:
if (response.isFailed())  is being checked at the start of the code block which will ensure failure is not null.